tinymce-basics/customizing-the-table-toolbar.html

```diff
@@ -0,0 +1,583 @@
+                                    buttonApi.setDisabled(state);
+                                }
+                            },
+                            {
+                                type: 'menuitem',
+                                text: 'Paste row after',
+                                icon: 'paste-row-after',
+                                onAction: () => {
+                                    menuActionManager.pasteRowAfter();
+                                },
+                                onSetup: (buttonApi) => {
+                                    // Check the clipboard for any rows and disable the menu item if there is none
+                                    // https://www.tiny.cloud/docs/plugins/table/#api
+                                    // https://www.tiny.cloud/docs/ui-components/typesoftoolbarbuttons/#api
+                                    const state = editor.plugins.table.getClipboardRows().length > 0 ? false : true;
+                                    buttonApi.setDisabled(state);
+                                }
+                            },
+                        ];
+                        callback(items);
+                    }
+                });
+
+                // Create a toolbar menu button to hold all row related options
+                // https://www.tiny.cloud/docs/ui-components/typesoftoolbarbuttons/#menubutton
+                editor.ui.registry.addMenuButton('columnoptions', {
+                    icon: 'table-column-options', // Insert the icon from the icon pack at the beginning of this file
+                    tooltip: 'Column options',
+                    fetch: (callback) => {
+                        var items = [
+                            {
+                                type: 'togglemenuitem',
+                                text: 'Header column',
+                                icon: 'table-left-header',
+                                onAction: () => {
+                                    menuActionManager.headerColumn();
+                                },
+                                onSetup: (buttonApi) => {
+                                    // Set the on/off state of the toggle button by query the type and return a boolean
+                                    const state = editor.queryCommandValue('mceTableColType') === 'th' ? true : false;
+                                    buttonApi.setActive(state);
+                                }
+                            },
+                            {
+                                type: 'separator',
+                            },
+                            {
+                                type: 'menuitem',
+                                text: 'Insert column before',
+                                icon: 'table-insert-column-before',
+                                shortcut: 'meta+alt+←', // Text displayed in the menuitem's shortcut label
```

Comment:
Some more where the shortcut text doesn't match what was registered?

Reply:
Fixed in an upcoming commit